rasa/core/policies/sklearn_policy.py

```diff
@@ -32,16 +42,29 @@
 class SklearnPolicy(Policy):
     """Use an sklearn classifier to train a policy."""
 
+    DEFAULT_MAX_HISTORY = 5
+
+    @staticmethod
+    def _standard_featurizer(
+        max_history: int = DEFAULT_MAX_HISTORY,
+    ) -> MaxHistoryTrackerFeaturizer:
+        # Sklearn policy always uses MaxHistoryTrackerFeaturizer
```

Comment:
I think the `why` would be more important as the code is pretty clear about the hardcoded `MaxHistoryTrackerFeaturizer`